Tools/WinMLRunner/Main.cpp

```diff
@@ -270,12 +276,12 @@ HRESULT EvaluateModel(
     // Add one more iteration if we ignore the first run
     uint32_t numIterations = args.NumIterations() + args.IgnoreFirstRun();
 
-    bool isGarbageData = !args.CsvPath().empty() || !args.ImagePath().empty();
+    bool isGarbageData = args.CsvPath().empty() || args.ImagePath().empty();
```

Comment:
Good catch! But should it be:
`args.CsvPath().empty() && args.ImagePath().empty(); `
If there is no CSV and also there is no Image provided, then we should use Garbage Data.

Reply:
Yes. That's right! I will correct it.